mantidproject/mantid
# Group workspace masking error in MaskDetectors

See the two scripts in the bug report for examples of how to reproduce the issue.

**To test:**
Two cases to check that didn't work before:
 - Check that a workspace (NOT a masked workspace) where
    - masked workspace has a number of spectra == the number of detectors in the input workspace
    - masked workspace has a number of spectra > the number of spectra in the input workspace
 - Check the above works when specifying a `StartWorkspaceIndex` and `EndWorkspaceIndex`
 - Code review.

Fixes #14793 .

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
